elytron/src/main/resources/org/wildfly/extension/elytron/LocalDescriptions.properties

```diff
@@ -888,6 +888,12 @@ elytron.filesystem-realm.hash-encoding=The string format for the password if it
 elytron.filesystem-realm.hash-charset=The character set to use when converting the password string to a byte array.
 elytron.filesystem-realm.credential-store=The reference to the credential store that contains the secret key to encrypt and decrypt the realm.
 elytron.filesystem-realm.secret-key=The alias of the secret key to encrypt and decrypt the realm.
+elytron.filesystem-realm.key-store=The reference to the key store that contains the key pair to perform filesystem integrity checks.
+elytron.filesystem-realm.key-store-alias=The alias of the key store to perform filesystem integrity checks.
```

Comment:
"The alias of the key store to perform filesystem integrity checks" -> "The alias of the entry within the key store to use to verify integrity"